dnf-behave-tests/features/microdnf/repolist-disabled.feature

```diff
@@ -0,0 +1,40 @@
+@no_installroot
+@destructive
+Feature: Repolist when all repositories are disabled
+
+
+Background:
+  Given I delete file "/etc/dnf/dnf.conf"
+    And I delete file "/etc/yum.repos.d/*.repo" with globs
+    And I use repository "dnf-ci-fedora" with configuration
+        |key      | value |
+        | enabled | 0     |
+
+
+Scenario: Repolist without arguments
+   When I execute microdnf with args "repolist"
+   Then the exit code is 0
+    And stdout is empty
+
+
+Scenario: Repolist with "--enabled"
+   When I execute microdnf with args "repolist --enabled"
+   Then the exit code is 0
+    And stdout is empty
+
+
+Scenario: Repolist with "--disabled"
+   When I execute microdnf with args "repolist --disabled"
+   Then the exit code is 0
+    And stdout contains "dnf-ci-fedora\s+dnf-ci-fedora test repository$"
```

Comment:
Is the output variable in any way? I think not, so can you please use the `stdout is ...` step for all the scenarios?